ansible/roles/master/templates/controller-manager.kubeconfig.j2

```diff
@@ -5,7 +5,7 @@ preferences: {}
 clusters:
 - cluster:
     certificate-authority: {{ kube_cert_dir }}/ca.crt
-    server: https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}
+    server: https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}
```

Comment:
I don't think we can remove groups['masters'][0]. I believe you need to set the vip using a conditional. Ex> if masters length > 1, then set the vip. Make sense?